index.html

```diff
@@ -1,26 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+    	
     	<meta charset="utf-8">
-    	<meta name="description" content="Your description goes here">
-    	<meta name="keywords" content="one, two, three">
+    	<meta name="relaxr" content="piece of mind">
+    	<meta name="keywords" content="relaxr, assistant, app">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
+	
 	<body>
-		<div id="container">
+		
+		<div class="container1">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<h1 id="logo">Relaxr</h1>
+				<h2 id="tagline">Get piece of mind with a single tap</h2>
+				<p id="getit"> Get It Now</p>
```

Comment:
@mbhat25 This should either be an `<a href="#"></a>` and style it  or a `button` tag. `p` tags are meant for text
 🔨